contrib/config/config.yml

```diff
@@ -1,23 +1,26 @@
 COMMON: &common
-  MAX_CONTENT_LENGTH: 16793600 # 16 MB
-  LOGGING: True
-  LOG_FILE_PATH: '/var/log/ccweb/'
-  LOG_FILE: 'cchecker_web.log'
+  MAX_CONTENT_LENGTH: 16793600
+  LOGGING: true
```

Comment:
this is case insensitive? You changed from `True` to `true`, but all the other booleans here are `True`